src/core/components/operation.jsx

```diff
@@ -183,7 +184,11 @@ export default class Operation extends PureComponent {
                   </div> : null
               }
 
-            <div className={(!tryItOutEnabled || !response || !allowTryItOut) ? "execute-wrapper" : "btn-group"}>
+            <div
+              className={cx("execute-wrapper", {
+                "sui-btn-group": (!tryItOutEnabled || !response || !allowTryItOut)
```

Comment:
**Question**: this is a functionality change, it wasn't formerly possible to have `execute-wrapper` and `sui-btn-group` applied at the same time. Intentional?

Reply:
Yes - it was intentional, but maybe not the best change. I wanted to apply the styles used on `.execute wrapper` to the `btn-group` to avoid adding a padding default to a reusable css class. Maybe it is better to create a specific wrapper for the `sui-btn-group`?